apps/server/src/integrations/sentry/webhook-response.ts

```diff
@@ -0,0 +1,204 @@
+import { PrismaClient } from '@prisma/client';
+import {
+  createIssue,
+  search,
+  setAccessToken,
+  createLinkedIssue,
+} from '@tegonhq/sdk';
+
+const prisma = new PrismaClient();
+
+export const webhookResponse = async (eventBody: any) => {
+  if (eventBody.action === 'created') {
+    return {
+      grant_type: 'authorization_code',
+      code: eventBody.data.installation.code,
+    };
+  }
+
+  if (eventBody.action === 'get-teams') {
+    return getAllTeams(eventBody.installationId);
+  }
+
+  if (eventBody.action === 'create-issue') {
+    return createTegonIssue(eventBody);
+  }
+
+  if (eventBody.action === 'get-issues') {
+    return getTegonIssues(eventBody);
+  }
+
+  if (eventBody.action === 'link-issue') {
+    return createTegonLinkedIssue(eventBody);
+  }
+
+  return true;
+};
+
+async function getAllTeams(accountId: string): Promise<any> {
+  const integrationAccount = await prisma.integrationAccount.findFirst({
+    where: { accountId, deleted: null },
+    include: { workspace: true, integrationDefinition: true },
+  });
+
+  if (!integrationAccount) {
+    return null;
+  }
+
+  const workspaceId = integrationAccount.workspaceId;
+  const teams = await prisma.team.findMany({
+    where: { workspaceId },
+    select: { id: true, name: true },
+  });
+
+  return teams.map(({ name, id }) => ({ label: name, value: id }));
+}
+
+async function createTegonIssue(eventBody: any): Promise<any> {
+  const {
+    installationId: accountId,
+    fields,
+    webUrl,
+    project,
+    issueId,
+    actor,
+  } = eventBody;
+  const { teamId, title, description } = fields;
+
+  const integrationAccount = await prisma.integrationAccount.findFirst({
+    where: { accountId, deleted: null },
+    include: { workspace: true, integrationDefinition: true },
+  });
+
+  if (!integrationAccount) {
+    return null;
+  }
+
+  const personalAccessToken = await prisma.personalAccessToken.findFirst({
+    where: { workspaceId: integrationAccount.workspaceId, deleted: null },
```

Comment:
it'll give any first personal access token in that workspace, we need the token for that specific action. 

add `type: 'trigger', user: {username: 'sentry'}`